src/App.tsx

```diff
@@ -1,15 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [search, setSearch] = useState('');
+
+  const toOneCase = (str: string) => str.toLowerCase();
+
+  const searchFilter = (movie: Movie) => {
+    const { title, description } = movie;
+    const searchLowerCase = toOneCase(search.trim());
+
+    return toOneCase(title).includes(searchLowerCase)
```

Comment:
split this with variables like - `const formatedTitle = toUserCase(title)`
make function for check if str includes substr